functional/tests/utilities.mjs

```diff
@@ -24,7 +24,7 @@ import path from "path";
 
 let mfProcess;
 let startMf = "npm run dev";
-const mfStarted = "Listening on HTTP port";
+const mfStartedAndSdkLoaded = "Loaded core SDK from URL";
```

Comment:
again, you're not loading the "core 'SDK'", you're loading any/all open rpc messages in the unified file, which happen to be spread across both SDKs.

Reply:
This is because here we are waiting for the log message that gets generated after loading unified openrpc in fireboltOpenRpc.mjs .i.e:
`logger.info(Loaded ${sdkName} SDK from URL ${url});
`sdkName here refers to the name inside supportedSdks in config file, which in this case is 'core' which has the unifiedopenrpc url as manage is removed from supportedSdks
```
` `"supportedSdks": [

        {
            "name": "core",
            "cliFlag": null,
            "cliShortFlag": null,
            "url": "https://meta.rdkcentral.com/firebolt/api/",
            "enabled": true
        },
```